spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -1,369 +1,372 @@
-* [ ] ChainLog
-  * [ ] Increment ChainLog version based on update type
-    * [ ] Major -> New Vat (++.0.0)
-    * [ ] Minor -> Core Module (DSS) Update (e.g. Flapper) (0.++.0)
-    * [ ] Patch -> Collateral addition or addition/modification (0.0.++)
-* [ ] `addresses_mainnet.sol` matches spell code
+* IF external contracts calls present (Not SubDAOs, e.g. Starknet)
+  * [ ] Target Contract don't block spell execution
+  * [ ] External call is NOT delegate call
+  * [ ] Target Contract doesn't have permissions on the Vat
+  * [ ] Target Contract doesn't do anything untoward (e.g. interacting with unsafe contracts)
+  * [ ] Contracts deployed via `CREATE2` (e.g. if it looks like a vanity address) do not have `selfdestruct` in their code
+  * [ ] MCD Pause Proxy doesn't give any approvals
+  * [ ] All possible actions of the Target Contract are documented
+  * [ ] Target contract is not upgradable
+  * [ ] Target Contract is included in the ChainLog
+  * [ ] Test Coverage is comprehensive
+* IF spell interacts with ChainLog
+  * [ ] ChainLog version is incremented based on update type
+    * Major -> New Vat (++.0.0)
+    * Minor -> Core Module (DSS) Update (e.g. Flapper) (0.++.0)
+    * Patch -> Collateral addition or addition/modification (0.0.++)
+  * [ ] New addresses are added to the `addresses_mainnet.sol`
+  * [ ] Changes are tested via `testNewOrUpdatedChainlogValues`
 * [ ] Ensure every spell variable is declared as `public`/`internal`
-* [ ] Ensure `immutable` visibility is only used when fetching addresses from the `ChainLog` via `DssExecLib.getChangelogAddress` and `constant` is used instead for static addresses
+* [ ] Ensure `immutable` visibility is only used when fetching addresses from the `ChainLog` via `DssExecLib.getChangelogAddress(key)` and `constant` is used instead for static addresses
   * [ ] Fetch addresses as type `address` and wrap with `Like` suffix interfaces inline (when making calls) unless archive patterns permit otherwise (Such as `MKR`)
   * [ ] Use the [DssExecLib Core Address Helpers](https://github.com/makerdao/dss-exec-lib/blob/master/src/DssExecLib.sol#L166) where possible (e.g. `DssExecLib.vat()`)
   * [ ] Where addresses are fetched from the `ChainLog`, the variable name must match the value of the ChainLog key for that address (e.g. `MCD_VAT` rather than `vat`), except where the archive pattern differs from this pattern (e.g. MKR)
 * [ ] Spell actions match the corresponding Exec Doc
-* [ ] Tests
+* Tests
   * [ ] Ensure each spell action has sufficient test coverage
     _List actions for which coverage was checked here_
   * [ ] Ensure every test function is declared as public if enabled or private if disabled
   * [ ] Ensure that the `DssExecLib.address` file is not being modified by the spell PR
   * [ ] Check all CI tests are passing as at the latest commit
     _Insert most recent commit hash where CI was passing_
   * [ ] Check all tests are passing locally using `make test`
```

Comment:
I suggest invert these 2 lines:
```suggestion
```

Reply:
Is this comment is the same as this one? https://github.com/makerdao/pe-checklists/pull/18#discussion_r1455948449

Reply:
Yes, the PR UI didn't allow me to make a suggestion encompassing both lines.